src/NumbersToLetters.java

```diff
@@ -40,9 +40,6 @@ public static void main(String[] args) {
             } else {
```

Comment:
code lijkt erg goed en clean! wat ik nog zou overwegen om regel 37 en 38 te combineren in 1 print statement. System.out.println(translator.translator(inputScanner));  dit is toch weer een regel code minder.